sorts/bubble_sort.py

```diff
@@ -22,14 +22,15 @@ def bubble_sort(collection):
     True
     """
     length = len(collection)
-    for i in range(length-1):
-        swapped = False
+    sorted = False
```

Comment:
There is really nothing wrong with using __break__.  It is part of the language.

__sorted__ is a builtin in Python so shadowing it with a variable name is a bad idea.